docs/observability/how_to_guides/tracing/trace_with_opentelemetry.mdx

```diff
@@ -0,0 +1,175 @@
+---
+sidebar_position: 17
+---
+
+import { ConfigureSDKEnvironmentCodeTabs } from "@site/src/components/QuickStart";
+import {
+  CodeTabs,
+  typescript,
+} from "@site/src/components/InstructionsWithCode";
+
+# Trace with OpenTelemetry
+
+LangSmith can now accept traces from from OpenTelemetry based clients.
+This guide will walk through examples on how to achieve this.
+
+## Logging Traces with a basic OpenTelemetry client
+
+This first section covers how to use a standard OpenTelemetry client to log traces to LangSmith.
+
+### 0. Installation
+
+Install the OpenTelemetry SDK, OpenTelemetry exporter packages, as well as the OpenAI package:
+
+```bash
+pip install openai
+pip install opentelemetry-sdk
+pip install opentelemetry-exporter-otlp
+```
+
+### 1. Configure your environment
+
+Setup environment variables for the endpoint, substitute your specific values:
+
+```bash
+export OTEL_OTEL_EXPORTER_OTLP_HEADERS=https://api.smith.langchain.com/otel
```

Comment:
Is this correct, why does it have OTEL twice?

Reply:
Oh - this is a typo